code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -152,6 +152,46 @@
 		if(spent)
 			. += span_cult("This shard is spent; it is now just a creepy rock.")
 
+/obj/item/soulstone/examine_more(mob/user)
+	. = ..()
+	if(!isliving(user))
```

Comment:
```suggestion
	if(!isliving(user) || isnull(user.mind))
```

Reply:
do mindless living mobs even have the capacity to examine atoms?

Reply:
Yep

Reply:
but ghosts are `/mob/dead/observer`s

they're deads, not livings

Reply:
Tru. Best be safe though anyways